Docs/Book/Cookbook_v2.rst

```diff
@@ -0,0 +1,1009 @@
+RDKit Cookbook v2
+%%%%%%%%%%%%%%%%%%%%
+
+.. sectionauthor:: Vincent F. Scalfani <vfscalfani@ua.edu>
+
+Introduction
+**************
+
+What is this?
+===============
+
+This document provides example recipes of how to carry out particular tasks using the RDKit functionality
+from Python. The contents have been contributed by the RDKit community, tested with the latest 
+RDKit release, and then compiled into this document. Note that this document is the second 
+iteration of the :doc:`Cookbook` (i.e., v2). The old Cookbook written in Markdown is no longer 
+maintained, but is available in prior RDKit releases for reference. The RDKit Cookbook v2 
+is written in reStructuredText, which supports Sphinx doctests, allowing for easier 
+validation and maintenance of the RDKit Cookbook v2 code examples, where appropriate. 
+
+What gets included?
+=====================
+
+The examples included come from various online sources such as blogs, shared gists, and 
+the RDKit mailing lists. Generally, only minimal editing is added to the examples for 
+formatting consistency and to incorporate the doctests. We have made a conscious effort 
+to appropriately credit the original source and authors. One of the first priorities of this
+document is to compile useful **short** examples shared on the RDKit mailing lists, as 
+these can be difficult to discover. It will take some time, but we hope to expand this 
+document into 100s of examples. As the document grows, it may make sense to prioritize 
+examples included in the RDKit Cookbook v2 based on community demand.
+
+Feedback
+=========
+
+If you have suggestions for how to improve the Cookbook v2 and/or examples you would like 
+included, please contribute directly in the source document (the .rst file). The Index ID# 
+is simply a way to track Cookbook entries and images. New additions are sequentially index numbered. 
```

Comment:
Hi @christopheryoung. Thanks so much for the review, it was very helpful. I intended for the latter. I agree, it is not ideal, but I'm not sure how else to do this, since arranging by topic makes the most sense to me. I went ahead and added a note at the top to help with tracking the index number.